templates/frontend/objects/article_details.tpl

```diff
@@ -432,6 +447,22 @@
 				</div>
 			{/if}
 
+			<div class="item">
+				<section class="sub_item">
```

Comment:
I would suggest to  make this as vue component as well. Including the scrolling action.

Encapsulating it to the vue.js component will make it easier to spread it to other themes, which has often its own templates.